.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+name: release
+
+on:
+  release:
+    types: [published]
+
+permissions:
+  contents: read
+  
+jobs:
+
+  release:
+    name: Release Signing
+    runs-on: ubuntu-latest
+    env:
+       RELEASE_TAG: ${{ github.ref_name }}
+    permissions:
+      contents: write
+      id-token: write
+      repository-projects: write
+       
+    steps:
+      - name: Sync Repository
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Create Archive Name
+        run: echo "MATERIALX_ARCHIVE=MaterialX-${RELEASE_TAG//v}" >> $GITHUB_ENV
+
+      - name: Generate Archives
+        run: |
+          git archive --prefix ${MATERIALX_ARCHIVE}/ --output ${MATERIALX_ARCHIVE}.zip ${RELEASE_TAG}
+          git archive --prefix ${MATERIALX_ARCHIVE}/ --output ${MATERIALX_ARCHIVE}.tar.gz ${RELEASE_TAG}
```

Comment:
@jstone-lucasfilm This is not needed anymore with sigstore/gh-action-sigstore-python v3. I created a PR in there to allow our use case and it was released in v3. You just need `release-signing-artifacts: true` and you can skip `inputs`.